matlab/+BalanceLaws/compute_numerical_solution.m

```diff
@@ -4,12 +4,18 @@
 
 % Takes the initialized fields and advances the solution in time
 
-global I_Mesh I_TI I_BalanceLaws I_Tech I_RunOps I_ConservedVars
+global I_Mesh I_TI I_BalanceLaws I_Tech I_RunOps I_Results
 
 if strcmp(I_Tech('REAL'),'float')
     current_time = single(zeros(2));
+    components = single(zeros(2));
+    norm2_output = single(zeros(1, I_Tech('num_groups')));
```

Comment:
Since we can use both the $L^2$ and the $L^\infty$ norm, we should change the name of `norm2_output`.